File: FibEven.js
/* Simple while loop for project2*/

var first = 1;
var second = 2;
var fib = 0;
var fibEven = 0;

while (first < 4000000) {

  if(first % 1 === 0) {
    fibEven += first;
  }
  fib = first + second;
  first = second;
  second = fib;

}
 
 console.log(fibEven);

File: FibEven.rb
$first = 1
$second = 2
$fib = 0
$fibeven = 0

for $first in 1...100
	if $first % 1 == 0
		$fibeven += $first
	end
  $fib = $first + $second
  $first = $second
  $second = $fib
end

puts $fibeven